README.md

```diff
@@ -62,7 +62,9 @@ config.ini :
 action=create
 u=http://localhost/cgi-bin/csaf_provider.go
 ```
-
+## csaf_checker
+csaf_checker is a tool for testing the CSAF-standards on the csaf_providers.  
```

Comment:
The csaf_checker (which probably should be named provider checker - at least in the human readable doc when describing its purpose) should not only work with this specific implementation `csaf_provider` but with any CSAF provider
```suggestion
csaf_checker is a tool for testing a CSAF trusted provider according to [Section 7 of the CSAF standard](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#7-distributing-csaf-documents).